Week3/homework/js-exercises/Project_CreditCardValidator.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+function validNumber(num) {
+    
+    const sumDigits = str => { //This function sums the digits of the credit card number in the string. Below I used an if statement to make sure that sum of the digits is greater than 16.
+        let sum = 0;
+        for (let char of str) {
+          sum += +char;
+        }
+        return sum
+      };
+
+      function allSameDigits(str) {
+        return /^(\d)\1+$/.test(str.replace(/[^\d]/g,'')); //I used a regex function to check whether all digits are same. Is that okay to use a regex (I kind of feel like cheating)? 
+    }
```

Comment:
Any reason of following a different convention for creating function here as compared to arrow function used above?

Reply:
As far as using regex is concerned, I would suggest you look into a new feature called `Sets`(https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) in JavaScript and see if you can use it achieve the desired result. If not, using regex is also fine as long as you understand how it is working and not simply copy pasting it from google 🙂

Reply:
Wow, I like Sets feature :). 

For using different conventions for the function: I will try to follow a uniform style from now on.